images/openshift-applier/root/usr/local/bin/run

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
-DEFAULT_INV=$HOME/test/inventory
+DEFAULT_INV=/etc/ansible
```

Comment:
This old path is "broken" as-is, but don't think changing it to `/etc/ansible` is the correct thing to do. The intent here was to just run the tests if nothing else was specified, and the tests live in the `tests` area of the repo. Maybe just leave this as-is and we can do a follow-up to correct this, unless you want to include it here?

Reply:
ok. I think I understand what you're saying but I don't think I know what the solution is.

Reply:
No worries. If we can address the php/httpd comment, we can merge as-is with an open issue to address the default inventory/tests.